eudemo/eudemo/reactor.py

```diff
@@ -71,10 +68,10 @@
 from eudemo.ivc.divertor_silhouette import Divertor
 from eudemo.maintenance.upper_port import UpperPortDesigner
 from eudemo.params import EUDEMOReactorParams
-from eudemo.pf_coils import PFCoil, PFCoilsDesigner
+from eudemo.pf_coils import PFCoil, PFCoilsDesigner, build_pf_coils_component
 from eudemo.power_cycle import SteadyStatePowerCycleSolver
 from eudemo.radial_build import radial_build
-from eudemo.tf_coils import TFCoil, TFCoilBuilder, TFCoilDesigner
+from eudemo.tf_coils.tf_coils import TFCoil, TFCoilBuilder, TFCoilDesigner
```

Comment:
maybe put these three imports into `tf_coils.__init__.py` so we can import from eudemo.tfcoils?